src/actions/mixins/store_breeze.cr

```diff
@@ -0,0 +1,36 @@
+class Fiber
+  property breeze_request : BreezeRequest?
+end
+
+module StoreBreeze
+  macro included
+    before store_breeze_request
+    after store_breeze_response
+  end
+
+  def store_breeze_request
+    req = BreezeRequest::SaveOperation.create!(
+      path: request.resource,
+      method: request.method,
+      action: self.class.name,
+      body: request.body.try(&.to_s),
+      parsed_params: JSON.parse(params.to_h.to_json),
+      session: JSON.parse(session.to_json),
+      headers: JSON.parse(request.headers.to_h.to_json)
+    )
+    Log.dexter.debug { {debug_at: Breeze::Requests::Show.url(req.id)} }
+    Fiber.current.breeze_request = req
```

Comment:
This is stored in the Fiber so other stuff can access it. For example we can associate queries and pipes with the request